Fedramp lead form

mike421453 opened the pull request:
Adding staging capability and then a slew of things for the Fedramp lead form for [this Issue here](https://github.com/github/marketing-operations/issues/535).

Commit:
adding staging capability and documentation to readme

Commit:
replace a bunch of URLs with relative for staging capability

Commit:
Eloqua form pages

Commit:
bunch of JS for Eloqua forms and Recaptcha

Commit:
Form CSS added

Commit:
Fedramp logos

Commit:
NEW fedramp pages

Commit:
updates to home using relative URLs for staging and Fedramp banner

mike421453 commented:
... working through Travis CI check failures now ...

jbjonesjr commented:
@mike421453 i can help you with those. There all likely in my bucket

mike421453 commented:
@sophshep - whole bunch of stuff going on here.  Not sure whose eyes are best for it, but you replied to the MOPS issue, so... !

mike421453 commented:
thanks @jbjonesjr !!

sophshep commented:
Thanks @mike421453! My plan is to work with @kimestoesta on the styling of things. We'll jump onto this branch once she's back from Ireland next week.

jbjonesjr committed:
Merge branch 'gh-pages' into fedramp-lead-form

jbjonesjr commented:
@mike421453 I've done my thing. failures are now back in your court.

jbjonesjr committed:
Merge branch 'gh-pages' into fedramp-lead-form

Commit:
alt tag in FedRAMP logo

Commit:
Merge branch 'fedramp-lead-form' of github.com:github/government.github.com into fedramp-lead-form

mike421453 committed:
undo ruby version accidental change

Commit:
adding permalinks to fedramp pages front-matter

Commit:
sans indexer slash

Commit:
re-indexifying permalinks for fedramp pages

Commit:
indexifying homepage link to fedramp

nschonni commented on _includes/header.html:
```diff
@@ -4,8 +4,8 @@
-  <link rel="shortcut icon" href="/assets/img/favicon.ico" type="image/x-icon">
-  <link rel="stylesheet" href="/assets/css/style.css">
+  <link rel="shortcut icon" href="{{ "/assets/img/favicon.ico" | relative_url }}" type="image/x-icon">
+  <link rel="stylesheet" type="text/css" href="{{ "/assets/css/style.css" | relative_url }}">
```
No `type` stuff for HTML5 (edit: ... for the default values text/css on stylesheet)

mike421453 replied:
Hi @nschonni - yes they are - I tried to find most/all of them and update.

I am still hitting some errors in TravisCI
```
- ./_site/fedramp-confirmation/index.html
  *  External link https://government.github.com/fedramp-confirmation/ failed: 404 No error
- ./_site/fedramp/index.html
  *  External link https://government.github.com/fedramp/ failed: 404 No error
```

and I am not sure how to remedy as those links look like they should be ok.

I kind of suspect it ties back to these lines in config.yml
```
url: https://government.github.com
relative_permalinks: false
```
but not really sure.  I am not a Jekyll expert by any means.  The resources site, which functions similarly, does not have those in the config.

cc: @jbjonesjr @sophshep

jbjonesjr replied:
@mike421453 yeah, so travis makes sure all links on the site are valid.  Since those two links (fedramp-confirmation and /fedramp) return `404` right now, travis is failing the build.  You can ignore those errors, especially if your PR is going to create those pages.

mike421453 replied:
ah ok great, thank you @jbjonesjr !

There will be more small changes from marketing so I'll wait for merge request - but that is great news.

nschonni reviewed (commented):
I'm guessing all the extra `relative_url` stuff on root links needed for the staging version

Commit:
marketing content changes

Commit:
FedRAMP Styling

Moved fed ramp banner out of home into index and added primer stylizing

Commit:
FedRamp Banner style and copy

Changed copy requested here:
https://github.com/github/creative/issues/1756#issuecomment-374384716

mike421453 commented:
Hi @kimestoesta - I staged your [changes here](https://pages.ghe.io/government/staging/).

@m0rgane - latest ...

If you want to do that, just run this in shell:

```
➜  script/stage staging
```

Commit:
Form Page Styling

Updated styling for the FedRAMP form page

kimestoesta commented:
@mike421453 @m0rgane made the last styling changes on my side. Good to ship from the design side.

Also I don't have a ghe account so I can't stage it. if you could do that I'd appreciate it 🙇‍♀️

mike421453 commented:
hi @kimestoesta - changes are all staged:
https://pages.ghe.io/government/staging/

@jbjonesjr - I am unable to do the merge in this repo with the outstanding TravisCI failures - can you do it?

jbjonesjr commented:
@mike421453 some of these i can fix, some will be fixed when you deploy your branch. let me fix the stuff I own and merge it in.

jbjonesjr committed:
Merge branch 'gh-pages' into fedramp-lead-form

jbjonesjr commented:
### Failed Build because of missing pages on the production site
![image](https://user-images.githubusercontent.com/533967/38595140-f806e65e-3d17-11e8-8f26-ca7113ed6b2c.png)

### Solution
![image](https://user-images.githubusercontent.com/533967/38595158-151b2c32-3d18-11e8-9a8f-6c6e206afde7.png)

This PR adds the files to the production site that were failing in the tests.

@mike421453 , feel free to admin-merge when you're ready.

mike421453 commented:
![screen shot 2018-04-11 at 10 12 21 am](https://user-images.githubusercontent.com/6828179/38622303-dc2ba1be-3d70-11e8-8cb6-92c1b3c56a83.png)

@jbjonesjr  I am not sure I have the ability to admin-merge this repo?

jbjonesjr commented:
@mike421453 got it! done!

mike421453 commented:
thank you so much!! woooo!
